cluster-autoscaler/core/scale_down.go

```diff
@@ -70,10 +70,10 @@ const (
 type NodeDeleteStatus struct {
 	sync.Mutex
 	deleteInProgress bool
-	// A map of node delete results by node name. It contains nil if the delete was successful and an error otherwise.
-	// It's being constantly drained into ScaleDownStatus objects in order to notify the ScaleDownStatusProcessor that
-	// the node drain has ended or that an error occurred during the deletion process.
-	nodeDeleteResults map[string]error
+	// A map of node delete results by node name. It's being constantly drained into ScaleDownStatus
```

Comment:
the `drain` verb is overloaded in this description (and in logic). Could we rename the `DrainNodeDeleteResults` to `GetAndClearNodeDeleteResults` and avoid using drain for other thing than draining the pods from node?

Reply:
Good idea, done!